src/model/stock_predictor.py

```diff
@@ -0,0 +1,30 @@
+from typing import Optional
+
+import torch
+from torch import nn
+
+
+class StockPredictor(nn.Module):  # pylint: disable= too-few-public-methods
```

Comment:
`# pylint: disable= too-few-public-methods`

Does our pylint config still enforce that rule? If not, this can be removed.

Reply:
I wanted to check it, but it seems that checks don't work for this pull request anymore.

Reply:
"Our pylint doesn't enforce this rule anymore. I removed it.